pilot/pkg/bootstrap/server.go

```diff
@@ -565,14 +564,17 @@ func (s *Server) initIstiodAdminServer(args *PilotArgs, wh *inject.Webhook) erro
 		Addr:    args.ServerOptions.HTTPAddr,
 		Handler: s.httpMux,
 	}
-
+	securedDebug := true
+	if s.httpServer.Addr != "localhost:8080" {
+		securedDebug = false
```

Comment:
it would be good to log our decision here if we change the behavior

Reply:
fixed